examples/ffva/ffva.cmake

```diff
@@ -76,6 +76,7 @@ if(ENABLE_ALL_FFVA_PIPELINES)
 else()
     set(FFVA_PIPELINES_INT
         fixed_delay
+        empty
```

Comment:
What does `empty` do or mean in this context?  Does it have something to do with `fixed_delay`?

I have noticed that `FFVA_PIPELINES_UA` does not include `empty` unless `ENABLE_ALL_FFVA_PIPELINES` is on.  Is that intentional?

Reply:
`empty` means a pipeline without any component, it is different from a bypassed pipeline, as in the bypass case the components are included, but they are not used, meaning that they take unnecessary memory.

In the `FFVA_PIPELINES_UA`  we never use the `empty` pipeline, we actually use it only in the new example with the I2S audio to the intent engine. The `empty` pipeline can be enabled in the UA examples by using:

`cmake -B build --toolchain xmos_cmake_toolchain/xs3a.cmake -DENABLE_ALL_FFVA_PIPELINES=1`

Reply:
That makes sense.  Thanks for the explanation.